_pages/install/schematics.md

```diff
@@ -45,15 +45,15 @@ Another example is the following, which generates an application that is ready t
 ng add @spartacus/schematics --baseUrl https://spartacus-demo.eastus.cloudapp.azure.com:8443/ --baseSite=apparel-uk-spa,electronics-spa --currency=gbp,usd --language=uk,en --ssr
 ```
 
-### Additional Commands for core libraries and features
+### Additional Commands for Core Libraries and Features
 
 By default, the `ng add @spartacus/schematics` command adds only a basic configuration of Spartacus. The following is a description of the commands you can use to extend your application:
 
 - `ng g @spartacus/schematics:add-pwa` adds a Spartacus-specific PWA module.
 - `ng g @spartacus/schematics:add-ssr` adds the SSR configuration.
 - `ng g @spartacus/schematics:add-cms-component` generates a CMS component, and adds the CMS component mapping to the specified module, or to a newly-generated module, if no module is specified. For more information, see [CMS Component Schematic](#cms-component-schematic), below.
 
-### How Spartacus Schematics Work
+## How Spartacus Schematics Work
```

Comment:
I think this section should be nested under the `Adding Spartacus Core Libraries and Features to Your Angular Project` section, as it only applies to it.

Reply:
I had thought maybe that section could stand on its own, but I see your point and have made the fix according to your suggestion 🙂